components/Sidebar.tsx

```diff
@@ -43,6 +43,9 @@ const linkClass = (isActive: boolean, isDisabled: boolean = false) =>
       : "text-zinc-400"
   } hover:text-zinc-100 flex items-center space-x-3`;
 
+const canConnectToDapp = (version: version) =>
+  version === "0.3.0" || version === "0.3.1" || version === "0.3.2";
```

Comment:
can we put an abstracted function call `isSupportPoE` in `versioned/interface.ts`? 
then we just call something like `Versioned.isSupportPoE` ? so we don't need to go back here when adding the new version.